src/protocolsupport/ProtocolSupport.java

```diff
@@ -103,7 +103,7 @@ public void onLoad() {
 			Class.forName(MapColorRemapper.class.getName());
 			Class.forName(PESkin.class.getName());
 			ServerPlatform.get().inject();
-			peserver = new MCPEServer(2222);
+			peserver = new MCPEServer(19138);
```

Comment:
@Shevchik Nah man. Just testing and cannot register 2222 on my machine. If you want to merge this than ofc it will be changed back.